Spec/Tests/UtilsTests.swift

```diff
@@ -0,0 +1,42 @@
+import Ably
+import Nimble
+import XCTest
+
+class UtilsTests: XCTestCase {
+    
+    struct Constants {
+        static let backoffValues = [ 1.0, 4.0/3.0, 5.0/3.0, 2.0, 2.0 ]
+        static let delayValues = [ 13.5, 18.0, 22.5, 27.0, 27.0 ] // calculated for randomFraction = 0.5 (jitterCoeff = 0.9)
+    }
+    
+    // MARK: Backoff and jitter
+    
+    func testRetryDelayCalculationForRetryNumberWithConstantRandomSeeding() {
+        let initialRetryTimeout = 15.0
+        
+        ARTUtils.seedRandom(withNumber: 5)
+        let delays1 = (1...5).map { n in initialRetryTimeout * ARTUtils.backoffCoefficient(forRetryNumber: n) * ARTUtils.generateJitterCoefficient() }
```

Comment:
Should this and the test below not be consistent in how they access the n<sup>th</sup> backoff coefficient? Either using `Constants.backoffValues` or using `ARTUtils.backoffCoefficient(forRetryNumber:)`.

Reply:
Good catch, thanks. What would you use in both places, constants or method?

Reply:
It's the sort of thing that I'm never sure about when testing, to be honest. Do you have any thoughts on it?

Reply:
Maybe, since we're not testing the calculation of the backoff coefficient elsewhere, we should use the `Constants.backoffValues`. But as I say, keen to hear your thoughts.

Reply:
While cleaning all this up, I've decided to put those values in the comment and go with an actual function https://github.com/ably/ably-cocoa/pull/1520/commits/7b35df981bdb2e1cd61012a161f1072aa48d3ae3